django/db/models/sql/subqueries.py

```diff
@@ -215,7 +215,5 @@ class AggregateQuery(Query):
     compiler = 'SQLAggregateCompiler'
 
     def add_subquery(self, query, using):
-        self.subquery, self.sub_params = query.get_compiler(using).as_sql(
-            with_col_aliases=True,
-            subquery=True,
-        )
+        query.subquery = True
```

Comment:
Not sure, I think Simon pointed out the same thing and suggested to call `query.clone()` before modifying it. I think I tried keeping the `subquery` argument, but it had to be removed to fix a test: a1da85bcd5f8f70c98bde1b1c8f23cb3fa931254.